Fast Pair: Separate show/hide UI indications from advertising mode (#11887)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

* :ref:`peripheral_fast_pair` sample:

  * Updated by disabling the :kconfig:option:`CONFIG_BT_SETTINGS_CCC_LAZY_LOADING` Kconfig option as a workaround fix for the `Zephyr issue #61033`_.

>   * Added automatic switching to the Fast Pair not discoverable advertising mode with the hide UI indication instead of removing Fast Pair advertising payload when all bond slots are taken.

Reviewer: ```suggestion
  * Added automatic switching to the Fast Pair not discoverable advertising mode with the hide UI indication instead of removing the Fast Pair advertising payload when all bond slots are taken.
```
We could interchange the 2 bullet point. Added and then updated.

  * Updated by deleting reset in progress flag from settings storage instead of storing it as ``false`` on factory reset operation.
    This is done to ensure that no Fast Pair data is left in the settings storage after the factory reset.

>   * Changed :c:struct:`bt_fast_pair_adv_config` to separate advertising mode from show/hide UI indication advertising info.

Reviewer 2: ```suggestion
  * Changed the :c:struct:`bt_fast_pair_adv_config` structure and the :c:enum:`bt_fast_pair_adv_mode` enumerator to separate advertising mode from show/hide UI indication advertising info.
```

>   * Changed the :c:struct:`bt_fast_pair_adv_config` structure and the :c:enum:`bt_fast_pair_adv_mode` enumerator to separate advertising mode from show/hide UI indication advertising info.

Reviewer: ```suggestion
  * Changed the :c:struct:`bt_fast_pair_adv_config` structure and the :c:enum:`bt_fast_pair_adv_mode` enumerator to separate advertising mode from show or hide UI indication advertising information.
```
